webpack.config.js

```diff
@@ -8,11 +8,10 @@ const html = fs.readFileSync(require.resolve('./tests/terra-dev-site/head.html')
 
 const devSiteConfig = (env = {}, argv = { p: false }) => ({
   // temporary fix to enable live reloading.
-  target: argv.p || argv.mode === 'production' ? 'browserslist' : 'web',
+  target: argv.p || argv.mode === 'production' ? 'browserslist' : 'browserslist',
```

Comment:
So the targeted environment will always be `browserslist`?

Reply:
oh whoops, this wasn't supposed to be checked in. you need to have browserlist set for webpack dev server to work in ie, but there is a bug in webpack dev server that breaks hot reloading when you have browserlist set... it's great.

Reply:
```suggestion
  target: argv.p || argv.mode === 'production' ? 'browserslist' : 'web',
```